org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -142,12 +147,30 @@ public SpaceMode DistanceSpaceMode
         public bool RejectZRollOff { get => rejectZRollOff; set => rejectZRollOff = value; }
 
         /// <summary>
-        ///  Speed for extending the moving button visuals when selected by a non-touch source.
+        /// Speed for extending the moving button visuals when selected by a non-touch source.
         /// </summary>
         [SerializeField]
         [Tooltip("Speed for extending the moving button visuals when selected by a non-touch source.")]
         private float extendSpeed = 0.5f;
 
+        /// <summary>
+        /// Array with Components (set in Editor) that will be enabled/disabled based on proximity and hovering.
+        /// </summary>
+        [field: SerializeField, Tooltip("Array with Components (set in Editor) that will be enabled/disabled based on proximity and hovering.")]
+        public Component[] ProximityHoveringDynamicComponents { get; set; } = null;
```

Comment:
I was under the impression that we decided to remove this property given @whebertML 's suggestion as well.  

Let's start with adding events only and add property later if needed.

Reply:
Aw :( ... I liked the idea of having the Component array because, imho, it simplifies making dynamic UIs with a tendency to improved rendering performance.  Such array empowers developers to quickly modify their scene and render UI components with rich visuals only when the user is about to interact with the associated UI elements and make a simple rendering when the user is not interacting with that part of the UI releasing GPU time for other tasks.  I was thinking on something like simplified UI rendering when the UI element is in the eye's Peripheral View and a rich rendering when the UI element is in the eye's Field of View.  Furthermore, such array, available in Editor would allow customers to quickly prototype and test their dynamic UIs for their specific needs.
On the other side, I do recognize you have more experience with MRTK3 framework and appreciate all you feedback and time you have invested in this PR, so I'll follow your lead; I'll remove the array next.

Reply:
Removed in latest commit.  Resolving this.